common-util/src/main/java/org/sunbird/common/responsecode/ResponseCode.java

```diff
@@ -534,6 +534,8 @@
       ResponseMessage.Message.FILE_ATTACHMENT_SIZE_NOT_CONFIGURED),
   emptyFile(ResponseMessage.Key.EMPTY_FILE, ResponseMessage.Message.EMPTY_FILE),
   invalidColumns(ResponseMessage.Key.INVALID_COLUMNS, ResponseMessage.Message.INVALID_COLUMNS),
+  invalidOrgLocations(
```

Comment:
Rename:
invalidOrgLocations to conflictingOrgLocations
INVALID_ORG_LOCATIONS to CONFLICTING_ORG_LOCATIONS.